### PR TITLE
Add Editorconfig

### PR DESCRIPTION
**Pull request by corvus-ch**
Adding Editorconfig as discussed in https://github.com/projectsyn/documentation/issues/40.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update tests.
- [x] Update the ./CHANGELOG.md.
- [x] Link this PR to related issues.

Documentation will be added to the syn documentation. See  https://github.com/projectsyn/documentation/issues/40.

### CODE REVIEW

**Review by simu — APPROVED** [review]

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,26 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
```
Comment (srueg) [inline .editorconfig:2]:
```suggestion
; More information at https://editorconfig.org
```

File: commodore/component-template/{{ cookiecutter.slug }}/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,26 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
```
Comment (srueg) [inline commodore/component-template/{{ cookiecutter.slug }}/.editorconfig:2]:
```suggestion
; More information at https://editorconfig.org
```

**Review by srueg — APPROVED** [review]

**Review by tobru — COMMENTED** [review]
Should the usage of .editorconfig be documented somewhere (README, Antora Docs)? I didn't know about it until now and I'm therefore asking to somewhere point out that this project makes use of it?

**Comment by corvus-ch** [comment]
> Should the usage of .editorconfig be documented somewhere (README, Antora Docs)? I didn't know about it until now and I'm therefore asking to somewhere point out that this project makes use of it?

Yes this should be documented. As hinted in this PRs description, this will go into the Project Syn documentation as part of a yet to be written style guide.

**Pull request merged**